src/constants.ts

```diff
@@ -1,136 +1,140 @@
-type Mapping = Record<number | string, string>;
+type Mapping = Record<number | string, string>
 
 export interface BaseConfig {
-  appId?: string;
+  appId?: string
 }
 
 /* eth series constants begin */
 
 export const ETH_CHAIN_ID_RPC_MAPPING: Mapping = {
   // BSC mainnet
-  56: "https://bsc-dataseed1.binance.org",
+  56: 'https://bsc-dataseed1.binance.org',
   // BSC testnet
-  97: "https://data-seed-prebsc-1-s1.binance.org:8545",
+  97: 'https://data-seed-prebsc-1-s1.binance.org:8545',
 
   // Polygon Mainnet
-  137: "https://rpc-mainnet.maticvigil.com/",
+  137: 'https://rpc-mainnet.maticvigil.com/',
   // Polygon Testnet
-  80001: "https://rpc-mumbai.matic.today/",
+  80001: 'https://rpc-mumbai.matic.today/',
 
   // Avalanche Mainnet
-  43114: "https://api.avax.network/ext/bc/C/rpc",
+  43114: 'https://api.avax.network/ext/bc/C/rpc',
   // Avalanche Fuji Testnet
-  43113: "https://api.avax-test.network/ext/bc/C/rpc",
+  43113: 'https://api.avax-test.network/ext/bc/C/rpc',
 
   // Arbitrum Testnet
-  421613: "https://endpoints.omniatech.io/v1/arbitrum/goerli/public",
+  421613: 'https://endpoints.omniatech.io/v1/arbitrum/goerli/public',
```

Comment:
Do we need to add the RPC mapping for the mainnet too?

Reply:
It's use for fallback, no needed

Reply:
Added

Reply:
thanks